ocaml/parsing/asttypes.mli

```diff
@@ -59,11 +59,6 @@ type const_layout =
 
 type label = string
 
-type arg_label =
```

Comment:
After some discussion, Nick and I agreed that this should stay right where it is (even if it is used only in the parser)